src/librbd/crypto/LoadRequest.h

```diff
@@ -0,0 +1,46 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: similar comment here w.r.t. should this just all be moved to the API layer.

Reply:
As above, I plan to extend this class to load encryption recursively for all ancestors reading from the image MD (asynchronously of course).